### DISCUSSION TITLE
Aluminum pole figure symmetry

### DISCUSSION THREAD

**Discussion by ghost** [mtex-toolbox/mtex#647] [Ask Anything]
Dear Developer,
I have derived my pole figures from EBSD data and want to compare it with experiments. The material is aluminium (m-3m). The experiment data doesn't distinguish upper and lower hemisphere. So I want to plot upper and lower hemisphere in one pole figure. I have tried the option 'antipodal', but it doesn't change anything, and I always have the pole figure as upper one...
I use the code: h = Miller({1,0,0},{0,1,1},{-1,1,1},ebsd('Aluminum').CS);
plotPDF(ebsd('Aluminum').orientations,h,'contourf','antipodal')
Is there anything missing in the code?
Thanks a lot in advance!

**Comment by kilir** [comment]
Hi,
I guess your aluminum data is 'm-3m' ? In that case 'antipodal' shouldn't do anything since, you should get identical (except that they are inverted) polefigures, the upper one is plotted as default in mtex. You can add 'complete' to check if both are identical.
Cheers,
Rüdiger
